test/aqua/operators/test_op_construction.py

```diff
@@ -221,6 +221,135 @@ def test_circuit_permute(self):
         c_op_id = c_op_perm.permute(perm)
         self.assertEqual(c_op, c_op_id)
 
+    def test_summed_op(self):
+        """ Test SummedOp """
+        sum_op: SummedOp = (X ^ X * 2) + (Y ^ Y)
+        self.assertEqual(sum_op.coeff, 1)
+        self.assertEqual(len(sum_op), 2)
+        self.assertEqual(str(sum_op.oplist[0].primitive), 'XX')
+        self.assertEqual(sum_op.oplist[0].coeff, 2)
+        self.assertEqual(str(sum_op.oplist[1].primitive), 'YY')
+        self.assertEqual(sum_op.oplist[1].coeff, 1)
```

Comment:
If we can, it would be good to avoid sequential assertions, since the test aborts immediately once a single test fails. 
Using `with self.subTest()` and `self.assertListEqual` this could get a bit more structure and the individual parts can fail independently. E.g.
 ```suggestion
        with self.subTest(msg='test construction'):
            self.assertEqual(sum_op.coeff, 1)
            self.assertEqual(len(sum_op), 2)
            self.assertListEqual([str(op.primitive) for op in sum_op.oplist], ['XX', 'YY'])
            self.assertListEqual([op.coeff for op in sum_op.oplist], [1, 2])
```

Reply:
Thanks. It makes sense. I will do it.

Reply:
Thanks a lot! 👍